Description: Use bash.
 .
 lifehash-cli (0.1.0-1) UNRELEASED; urgency=medium
 .
   * Initial release. (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: Nick Ochiel <nochiel@users.noreply.github.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2023-02-17

--- lifehash-cli-0.1.0.orig/build.sh
+++ lifehash-cli-0.1.0/build.sh
@@ -1,4 +1,4 @@
-#!/bin/zsh
+#!/bin/bash
 
 set +e
 
--- lifehash-cli-0.1.0.orig/src/build_deps.sh
+++ lifehash-cli-0.1.0/src/build_deps.sh
@@ -1,8 +1,9 @@
-#!/bin/zsh
-
+#!/bin/bash
 set +e
 
-git submodule update --init
+if [[ $(git submodule status | head -n1) == -* ]]; then
+    git submodule update --init
+fi
 
 DEPS_ROOT=${PWD}/deps
 BUILD_ROOT=${DEPS_ROOT}/build
